src/v1/sources/shopify/util.js

```diff
@@ -0,0 +1,306 @@
+/* eslint-disable no-param-reassign */
```

Comment:
If this code is duplicate for Shopify V0 lets import from there rather than duplicating it

Reply:
I thought migrating everything to v1 and then later removing all v0 code. Is it not the migration process?

Reply:
I think we are not immediately removing `v0` once we do that migration we can do the movement